### PR TITLE
1757 - Fix categorical labels

### PR DESCRIPTION
**Pull request by AchyutRatkanthiwar**
## Description
Trello ticket [#1757](https://trello.com/c/RFEdbe0T/1757-s-clean-workspace-16-add-validation-for-reconciliation-dataset)

This ticket is to add validations for reconciliation data within clean workplace data step. But while adding these validations, we realised the categorical label mapping had some gaps. this PR is to update that and a separate PR will be created to update the validations.

## Testing
- [x] Unit tests passing

## Reviewer checklist
- [ ] PR solves the Trello ticket requirements
- [ ] Outputs appear reasonable and understandable
- [ ] The overall approach is appropriate and not over-engineered
- [ ] No obvious scalability, performance or interdependency concerns
- [ ] Tests appropriately cover the main behaviour/edge cases
- [ ] Naming and structure are broadly understandable and consistent
- [ ] Docstrings are sufficient for future maintenance
- [ ] Docstrings cover non-obvious behaviour

### CODE REVIEW

**Review by MarionJHolloway — APPROVED** [review]

File: utils/column_values/categorical_column_values.py [hunk 1]
```diff
@@ -665,6 +666,7 @@ class ParentPermission(ColumnValues):
 
     parent_has_ownership: str = "Parent has ownership"
     workplace_has_ownership: str = "Workplace has ownership"
+    placeholder_label: str = "Placeholder label"
```
Comment (garypolzin) [inline utils/column_values/categorical_column_values.py]: can you remove this one and the reference to 3 please - looks to be a raw data issue as opposed to a missing label
